### PR TITLE
Update review_guide.html

### PR DESCRIPTION
**Pull request by atsay**
Updated guidelines, removed Guidelines for Rating section.

### CODE REVIEW

File: apps/pages/templates/pages/review_guide.html [hunk 1]
```diff
@@ -94,14 +83,12 @@ <h2 id="tips">{{ _('Some tips for writing a great review') }}</h2>
     <ul>
       <li>
         {% trans %}
-          Submit simple reviews such as "Great!", "wonderful", or "bad" with
-          no further explanation.
+          Submit one-word reviews such as "Great!", "wonderful", or "bad".
```
Comment (clouserw) [inline apps/pages/templates/pages/review_guide.html]: I'm bad about following the rule, but punctuation should be inside the quotes

**Comment by atsay** [comment]
Great, now spasovski is trolling me.

File: apps/pages/templates/pages/review_guide.html (hunk 1 above)
Comment (atsay) [inline apps/pages/templates/pages/review_guide.html]: Not always! http://www.quickanddirtytips.com/education/grammar/how-to-use-quotation-marks

**Comment by magopian** [comment]
Nice, thank you! Should I merge this now or are you waiting for other reviews (pun not intended)?

**Comment by atsay** [comment]
We're ready now--thanks!

**Pull request merged**